tests/phpunit/payment-tokens/class-wc-stripe-payment-tokens-test.php

```
@@ -440,4 +498,223 @@ public function test_get_account_saved_payment_methods_list_item( WC_Payment_Tok
+	 * When the user is not logged in, the tokens array must be returned unchanged
+	 * (no Stripe API call or gateway access is needed).
+	 */
+	public function test_woocommerce_get_customer_payment_tokens_returns_unchanged_when_not_logged_in(): void {
+		wp_set_current_user( 0 );
+
+		$initial_tokens = [ 'mock_token_key' => new WC_Payment_Token_CC() ];
+
+		$result = $this->stripe_payment_tokens->woocommerce_get_customer_payment_tokens( $initial_tokens, 1, WC_Stripe_UPE_Payment_Gateway::ID );
+
+		$this->assertSame( $initial_tokens, $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_woocommerce_get_customer_payment_tokens_returns_unchanged_when_not_logged_in is not named in camelCase. (undefined)

(CamelCaseMethodName)